src/utils/constants.ts

```diff
@@ -51,6 +51,7 @@ export const TABLET_SYMBOLS = {
     BlockStoreVolume: 'BV',
     Console: 'CN',
     TenantSlotBroker: 'TB',
+    BlockStoreDiskRegistry: 'BDR',
```

Comment:
Could you please use `EType` enum values from `/types/api/tablet` instead of string keys here? I think it will be more error safe:
```
[EType.OldTxProxy]: "P",
```